components/resources/brave_vpn_strings.grdp

```diff
@@ -367,6 +371,9 @@ Let's get you connected!
   <message name="IDS_BRAVE_VPN_MAIN_PANEL_SESSION_EXPIRED_PART_TITLE" desc="Title text for session expired panel">
     Session expired
   </message>
+  <message name="IDS_BRAVE_VPN_MAIN_PANEL_OUT_OF_CREDENTIALS_TITLE" desc="Title text for when user is out of credentials and they can't authenticate">
+    Can't authenticate to VPN
```

Comment:
sure this isn't missing a "the"?

"Can't authenticate to _the_ [virtual private] network"?